ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -747,17 +751,17 @@ def generate_from_vlan(self):
 
     def generate_ping_dut_lo(self):
         self.ping_dut_packets = []
-        dut_lo_ipv4 = self.test_params['lo_prefix'].split('/')[0]
+        dut_lo_ipv4 = self.lo_prefix.split('/')[0]
         for src_port in self.vlan_host_ping_map:
             src_addr = random.choice(self.vlan_host_ping_map[src_port].keys())
             src_mac = self.hex_to_mac(self.vlan_host_ping_map[src_port][src_addr])
-            packet = simple_icmp_packet(eth_src=src_mac,
-                                        eth_dst=self.dut_mac,
+            packet = simple_icmp_packet(eth_src=src_mac,,
```

Comment:
Extra `,` added. Surprising how this did not fail the call to `simple_icmp_packet`.

Reply:
Was supposed to be added to latest commit but missed, fixed.